src/style.css

```diff
@@ -1,3 +1,71 @@
+html {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  height: 60px;
```

Comment:
Avoid fixing container sizes unless it's a requirement. Let the content dictate the container's size to prevent overflow or accidental scroll bars.